tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -72,15 +72,11 @@ sub run {
 
     if (check_var('RUN_TEST_ONLY', 0)) {
         use_ssh_serial_console;
-        my $sut_ip = get_required_var('SUT_IP');
-
-        set_var('SERVER_IP', $sut_ip);
-        bmwqemu::save_vars();
 
         # Synchronize the server & agent node before setup
         barrier_wait('kubevirt_test_setup');
 
-        my $agent_ip = $self->get_var_from_child("AGENT_IP");
+        my $agent_ip = $self->get_var_from_child("SUT_IP");
         record_info('Agent IP', $agent_ip);
```

Comment:
You still refer to agent ip here.

Reply:
It's similar, it will get the value of SUT_IP from agent node, depends on which one is the child machine.